src/constants.rs

```diff
@@ -72,6 +72,7 @@ lazy_static! {
         options_limit: OptionsLimit::default(),
     };
     pub static ref TYPE_PRIORITIES: HashMap<&'static str, i32> = vec![
+        ("all", 5),
```

Comment:
Yes, this is needed when listing addon catalogs, since this type should be the first in the list, but actually is sorted after all the priority types.
![image](https://user-images.githubusercontent.com/39882951/235854766-8b726854-d507-43d3-807e-41cf659259b6.png)
It a small change so thought I'd include it here with a related the addon action as well.